packages/create-flex-plugin/templates/package.json

```diff
@@ -1,25 +1,25 @@
 {
-  "name": "{{pluginFileName}}",
```

Comment:
Why did you change this?

Reply:
It was a duplicate; literally, we were doing `config.pluginFileName = config.name` and so instead of adding another key to the interface, just decided to use `name` all the way.